README.md

```diff
@@ -68,6 +68,9 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 * [DFIR intro](https://medium.com/@sroberts/introduction-to-dfir-d35d5de4c180/) - By Scott J. Roberts.
 * [The Practice of Network Security Monitoring: Understanding Incident Detection and Response](http://www.amazon.com/gp/product/1593275099) - Richard Bejtlich's book on IR.
+* [Incident Response & Computer Forensics, Third Edition](https://www.amazon.com/Incident-Response-Computer-Forensics-Third/dp/0071798684/) - The definitive guide to incident response
```

Comment:
plz sort in category by alphabetical  order

Reply:
Done. :)